### DISCUSSION TITLE
如何使用pwa插件呢？我配置了后本地启动，没有出现安装按钮

### DISCUSSION THREAD

**Discussion by ChenSino** [vuepress/ecosystem#320] [Q&A] [answered]
我使用`npm i -D @vuepress/plugin-pwa@next`安装了pwa插件,然后再theme.ts的plugins下配置了：

![image](https://github.com/user-attachments/assets/e6824b82-a767-4119-bb1f-f1ceeee4ec3e)
然后启动，谷歌浏览器访问<http://localhost:8080>，没有出现安装按钮，求问我是少了什么配置吗？
我的版本如下：
![image](https://github.com/user-attachments/assets/d3893a60-ce06-496e-a3ca-5e1c67f4ac2b)

**Comment by Mister-Hope** [comment, marked as answer]
pwa要求必须使用https

**Reply by ChenSino** [reply]
> pwa要求必须使用https

ok，感谢感谢~
